core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -545,6 +558,16 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
     }
   }
 
+  protected def createScramAdminClient(scramMechanism: String, user: String, password: String): Admin = {
```

Comment:
can we create a common function in SaslSetup for this?  Seems to be repeated in a lot of tests.

Reply:
See above

Reply:
Ok, this now invokes a new method on SaslSetup().